File: src/main/java/com/sport/service/web/models/joint_training/JointTrainingResponse.java
package com.sport.service.web.models.joint_training;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class JointTrainingResponse {
    private Long id;
    private String title;
    private String description;
    @JsonFormat(pattern = "yyyy-MM-dd")
    private LocalDate date;
    @JsonFormat(pattern = "HH:mm")
    private LocalTime time;
    private String sportType;
    private String placeName;
    private String district;
    private String address;
    private String creatorName;
    private String phoneNumber;
    private String linkToChatWithCreator;
}

File: src/main/kotlin/com/sport/service/web/models/joint_training/JointTrainingResponse.kt
package com.sport.service.web.models.joint_training

import java.time.LocalDate
import java.time.LocalTime

data class JointTrainingResponse(
    val title: String,
    val description: String,
    val date: LocalDate,
    val time: LocalTime,
    val sportType: String,
    val placeName: String,
    val district: String,
    val address: String,
    val creatorName: String,
    val phoneNumber: String,
    val linkToChatWithCreator: String
)